src/hooks/useServiceContainer.ts

```diff
@@ -180,11 +177,9 @@ export const useServiceContainer = <T extends TServiceData, U extends TServicePa
         if (loadingCount.current <= 1) {
           // In a future React version (potentially in React 17) this could be removed as it will be default behavior
```

Comment:
Comments should be removed if `ReactDOM.unstable_batchedUpdates` is not used anymore.